pennylane/resource/resource.py

```diff
@@ -125,6 +219,231 @@ def resources(self) -> Resources:
         """
 
 
+def add_in_series(r1: Resources, r2: Resources) -> Resources:
+    r"""
+    Add two :class:`~.resource.Resources` assuming the circuits are executed in series.
```

Comment:
Make sure that there is a blank line after the first sentence. Otherwise, the docs will contain a paragraph instead of a line.

Reply:
Please apply this everywhere.